src/Form/MetadataDisplayForm.php

```diff
@@ -254,38 +296,77 @@ public static function ajaxPreview($form, FormStateInterface $form_state) {
           'mode' => 'application/json',
         ],
       ];
-      sort($used_vars);
-      $json_keys = array_keys($jsondata);
-      $data_json = array_map(function($key) {
-        return 'data.' . $key;
-      }, $json_keys);
-      $unused_vars = array_diff($data_json,$used_vars);
-      sort($unused_vars);
-      $used_rows = array_map(function($used) {
-        return [$used];
-      }, $used_vars);
-      $unused_rows = array_map(function($unused) {
-        return [$unused];
-      }, $unused_vars);
-      $var_table = [
-        '#type' => 'table',
-        '#header' => [t('Used Variables')],
-        '#rows' => $used_rows,
-        '#empty' => t('No content has been found.'),
-      ];
-      $json_table = [
-        '#type' => 'table',
-        '#header' => [t('Unused JSON keys')],
-        '#rows' => $unused_rows,
-        '#empty' => t('No content has been found.'),
-      ];
 
-      // Try to Ensure we're using the twig from user's input instead of the entity's
-      // default.
       try {
+        // Try to Ensure we're using the twig from user's input instead of the entity's
+        // default.
         $input = $form_state->getUserInput();
-        $entity->set('twig', $input['twig'][0], FALSE);
+        $preview_template = $input['twig'];
```

Comment:
Why the extra $preview_template = $input['twig']; and then $preview_template[0] ?

Reply:
I think I had used that variable to try something else at one point and forgot to change it back. Will fix.

Reply:
Addressed via 11c8f2f2b3e7496807fa06488e2eda109911a408.